steps/lifx-notifier/1.2.1/step.yml

```
@@ -0,0 +1,105 @@
+title: LIFX Notifier
+summary: Sends signals to your LIFX bulb during builds.
+description: |-
+  Add this step to your workflow when color change is required.
+
+  This step can be used at the start (notify build start) and at the end (notify success/failure) of your workflow.
+  If there's no need to notify about workflow start than adding step to the end of the workflow will be enough.
```

```suggestion
  If there's no need to notify about workflow start then adding step to the end of the workflow will be enough.
```
Anyway, this sentence does not add any effective information IMO.